bazel.sh

```diff
@@ -22,6 +22,7 @@
 # Arguments:
 #   1. bazel action (build or test, usually)
 #   2. BUILD target.
+#   3. Minimum Xcode version. E.g. "8" or "8.2.1"
```

Comment:
Does it make sense to have a default value equal to what is locally installed?
xcrun xcodebuild -version

Reply:
By default it'll run against every Xcode installation available, which I think may be preferable for this script. What do you think?

Reply:
SGTM.